### PR TITLE
Add 'apple-touch-icon' link tag to improve the lighthouse scores

### PR DESCRIPTION
**Pull request by reznord**
**What kind of change does this PR introduce?**
Bugfix

**Did you add tests for your changes?**
Yes

**Summary**
Adds 'apple-touch-icon' for improving the lighthouse audit score.

**Does this PR introduce a breaking change?**
No

**Other information**

Please paste the results of `preact info` here.
```
Environment Info:

  System:
    OS: macOS 10.15.3
    CPU: (8) x64 Intel(R) Core(TM) i5-8259U CPU @ 2.30GHz
  Binaries:
    Node: 12.16.1 - ~/.nvm/versions/node/v12.16.1/bin/node
    Yarn: 1.22.0 - /usr/local/bin/yarn
    npm: 6.13.4 - ~/.nvm/versions/node/v12.16.1/bin/npm
  Browsers:
    Chrome: 80.0.3987.122
    Firefox: 73.0.1
    Safari: 13.0.5
```

### CODE REVIEW

**Comment by prateekbh** [comment]
I dont know what is `size-plugin` supposed to do here? cc @developit

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,6 @@ tmp-*.json
 
 .idea/
 .vscode
+
+# auto generted while running tests
```
Comment (ForsakenHarmony) [inline .gitignore]:
```suggestion
# auto generated while running tests
```

**Comment by ForsakenHarmony** [comment]
Test failure seems unrelated

File: packages/cli/tests/create.test.js [hunk 2]
```diff
@@ -29,6 +29,15 @@ describe('preact create', () => {
 		expect(template.includes('twitter:card')).toEqual(true);
 	});
 
+	it(`should have 'apple-touch-icon' meta tag`, async () => {
```
Comment (prateekbh) [inline packages/cli/tests/create.test.js]: This test should probably. test more things in head rather than just an apple meta tag

**Pull request merged**